docs/faq.rst

```diff
@@ -91,6 +101,40 @@ The F5 ACI ServiceCenter Visibility tables have an option to select the **Partit
 Note: This is a new behavior in FASC v2.6 and above. The previous versions do show Common partition entries along with the selected partition entries.
 
 
+Visibility Dashboard
+--------------------
+
+**Q. Why is the BIG-IP Endpoint Details section on the Visibility Dashboard blank?** (Applicable to v2.7+)
+
+BIG-IP Endpoint Details section on the Visibility Dashboard may not display information due to the MAC address table getting flushed on the BIG-IP.
+
+Workaround: Send an ARP request to the host or check the connectivity with the host using ping command.
+
+------
+
+**Q. Why is the 'Interface' column blank in the BIG-IP endpoint details section on the Visibility Dashboard?** (Applicable to v2.7+)
+
+The **Interface** column in the BIG-IP Endpoint Details section on the Visibility Dashboard will be blank for vCMP guests since the behavior of the vCMP Guest BIG-IP also is the same; i.e. no interface information for VLANs.
+
+------
+
+**Q. Why are the pool members displayed on Visibility VIP table, and the pool members displayed on the Visibility VIP dashboard not the same?**
+
+Visibility VIP table displays the pool members from a BIG-IP VIP, only if they are also present on the APIC. However, the Visibility dashboard shows all the pool members (and associated stats) that are present on the BIG-IP even if they may or may not be present on the APIC. Hence both the outputs may be different.
+
+------
+
+**Q. What does the field 'Route Domain' on the Visibility Dashboard indicate?**
+
+It displays the **default route domain** for the partition to which the Virtual Server (VIP) or Node belongs to.
+
+------
+
+**Q. Why don't I see the scrollbar for 'View Logs' window on the Visibility Dashboard?**
+
+If you encounter this issue, please zoom out your web browser window by clicking the **Ctrl** key, and then press the **-** (i.e. minus key).
```

Comment:
...this issue, use the 'zoom out' option on your web browser. For example, on Windows, hold the **Ctrl** key, and then click **-** (the dash/minus key).